hera_qm/firstcal_metrics.py

```diff
@@ -135,7 +135,7 @@ def plot_stds(metrics, fname=None, ax=None, xaxis='ant', kwargs={}, save=False):
         ax.set_xlim(-1, Nants)
         ax.set_xticks(range(Nants))
         ax.set_xticklabels(metrics['ants'])
-        [t.set_rotation(20) for t in ax.get_xticklabels()]
+        [label.set_rotation(20) for label in ax.get_xticklabels()]
```

Comment:
Curious this list comprehension was left. Can we change this to a for loop to match the plotting code in omnical_metrics.

Reply:
Yeah, I mostly got tired. I'll fix it